client/templates/kontakt/kontakt.html

```diff
@@ -0,0 +1,47 @@
+<template name="kontakt">
+
+  <div class="menu-sub-view">
+    <div class="houseoffam-logo-mobile">
+      <a href="/" class="menu-nav-hover"> HOUSE OF FAM</a>
+    </div>
+    <div class="menu-nav-sub-wrapper">
+      <div class="menu-nav-sub nettbutikk-sub">
+        <a href="http://fam-irvoll.myshopify.com/collections/frontpage" class="menu-nav-hover"> NETTBUTIKK </a>
+      </div>
+      <div class="menu-nav-sub kontakt-sub">
+        <a href="/kontakt" class="menu-nav-hover">
+          KONTAKT
+        </a>
+      </div>
+      <div class="menu-nav-sub om-sub">
+        <a href="/om" class="menu-nav-hover">
+        OM
+        </a>
+      </div>
+    </div>
+
+    <div class="houseoffam-logo-sub">
+      <a href="/">
+      HOUSE OF FAM
+      </a>
+    </div>
+  </div>
+
+<div class="post">
+
+    <div class="kontaktmanager">
+
+      <div id="manager">
+      <p> All kontakt sendes til </p>
+    </div>
+
+     <br>
+
+    <div id="managername">
+      Ryan Ludlow
+    </div>
+     <a href="mailto:ryan@brandentevents.com"> <div id="manageremail"> ryan@brandentevents.com </div> </a>
+    </div>
+
+  </div> <!-- post -->
+</template>
```

Comment:
Yes I now. You can remove it if you like. The menu is different from if its in the main page or in a sub page. If its in admin or public. But if you want to make the layout now I can go in and modify it to the correct design later together with the other design editing. 


> 19. mai 2016 kl. 16.33 skrev Sebastian Sastre <notifications@github.com>:
> 
> In client/templates/kontakt/kontakt.html <https://github.com/dagvonkr/hof3/pull/11#discussion_r63889101>:
> 
> > +    <div class="kontaktmanager">
> > +
> > +      <div id="manager">
> > +      <p> All kontakt sendes til </p>
> > +    </div>
> > +
> > +     <br>
> > +
> > +    <div id="managername">
> > +      Ryan Ludlow
> > +    </div>
> > +     <a href="mailto:ryan@brandentevents.com"> <div id="manageremail"> ryan@brandentevents.com </div> </a>
> > +    </div>
> > +
> > +  </div> <!-- post -->
> > +</template>
> All these lines repeating the header are redundant (violating DRY). The way to do it is to tell the router that it should wrap this route using a the blogLayout @dagvonkr <https://github.com/dagvonkr> is not good to merge. Do you want to correct this yourself or I do it?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/dagvonkr/hof3/pull/11/files/f1a0b514e1ddbd8dbcd90b014e189a7cdea7bcb6#r63889101>